x-pack/plugins/cases/public/containers/configure/utils.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { CasesConfigurationUI } from '../types';
+import { ConnectorTypes } from '../../../common';
+
+export const initialConfiguration: CasesConfigurationUI = {
+  closureType: 'close-by-user',
+  connector: {
+    fields: null,
+    id: 'none',
+    name: 'none',
+    type: ConnectorTypes.none,
+  },
+  customFields: [],
+  mappings: [],
+  version: '',
+  id: '',
+  owner: '',
+};
+
+export const getConfigurationByOwner = ({
+  configurations,
+  owner,
+}: {
+  configurations: CasesConfigurationUI[] | null;
+  owner: string;
```

Comment:
I think we should take into account a situation where the owner is not defined. For example, if I do `owner: owner[0]` and the `owner` is an empty array, then it will be `undefined`.

Reply:
is the `initialConfiguration` the expected value in that situation?

Reply:
Yes, I think this should be our fallback.